### DISCUSSION TITLE
Maintenance glossarium

### DISCUSSION THREAD

**Discussion by quachpas** [typst-community/org#11] [General]
Hello!
I've seen the discussion on Discord about about @slashformotion looking for maintainers for glossarium, albeit too late. I have contributed [previously](https://github.com/typst-community/glossarium/pull/37) to glossarium and have a [WIP PR pending](https://github.com/typst-community/glossarium/pull/52) for Typst v0.12.0. I was waiting for slash's input, but now I realize he might not answer at all!
Who should I discuss the PR with? Also, please know that if you do need an active maintainer, I do have some free time on the side to work on glossarium, even if I'm not the best at scripting!
Best

**Comment by huwaireb** [comment]
The repository is currently unmaintained. The purpose of @slashformotion's transfer to the typst-community organization was to find another maintainer. If you'd like to take over, I'd be happy to grant maintainer permissions over the repository.

**Reply by quachpas** [reply]
Sure! I'll be glad to take over. Thanks for the quick answer

**Comment by huwaireb** [comment]
Maintainer-ship rights granted

**Comment by slashformotion** [comment]
Hello all, as I said in my message on discord, I will not continue to work on glossarium. I am glad that the project can continue here.
